libbeat/_meta/fields.common.yml

```diff
@@ -5,10 +5,9 @@
     Contains common beat fields available in all event types.
   fields:
     - name: agent.hostname
-      type: keyword
-      description: >
-        Deprecated - use agent.name or agent.id to identify an agent.
-        Hostname of the agent.
+      type: alias
+      path: agent.name
+      migration: true
```

Comment:
I wonder if we want to keep an alias, is this used by any UI?

Reply:
There is at least one place using `agent.hostname` in Kibana. There may be more instances. So I'll remove the migration tag so that the alias always exists (at least until we can confirm all of Kibana is updated in 8.0).

I'll ask the uptime team if they can update 8.0 to use `agent.name`.

https://github.com/elastic/kibana/blob/523926f0a49a5bd951ed0e056610b049482adbca/x-pack/plugins/uptime/common/runtime_types/ping/ping.ts#L58

Reply:
cc @andrewvc